lib/enum_weasel/get_model_enums.rb

```diff
@@ -0,0 +1,23 @@
+module EnumWeasel
+  class GetModelEnums
+
+    def call
+     model_enums= {}
+
+     Rails.application.eager_load!
+      ActiveRecord::Base.send(:subclasses).map(&:name).each { |model|
+        unless model.constantize.defined_enums.empty?
```

Comment:
Maybe this can be broken up into methods like

`def has_enum?`

and

`def model_enums`

or something like that.